.github/workflows/basic.yml

```diff
@@ -189,11 +194,15 @@ jobs:
           ctest --output-on-failure
 
       - name: Test make install
+        # Run only when the example is built
```

Comment:
Is this because the file that checks the installation contains a fixed list of files? If so, could you add a TODO here + submit an issue about making install checks more smart?